src/applications/hca/tests/components/FormFields/DependentList.unit.spec.js

```diff
@@ -48,33 +48,41 @@ describe('hca <DependentList>', () => {
       const tiles = view.container.querySelectorAll(
         '.hca-dependent-list--tile',
       );
-      const selector = tiles[0].querySelector(
-        '[data-testid="hca-dependent-tile-name"]',
-      );
+      const selectors = {
+        name: tiles[0].querySelector('[data-testid="hca-dependent-tile-name"]'),
+        relationship: tiles[0].querySelector(
+          '[data-testid="hca-dependent-tile-relationship"]',
+        ),
```

Comment:
You could use `import { $ } from 'platform/forms-system/src/js/utilities/ui';`, then this might be a bit easier to read: `$('[data-testid="hca-dependent-tile-relationship"]', titles[0]),`